v/vertx-micrometer-metrics/vertx-micrometer-metrics_ubi8.4.sh

```diff
@@ -0,0 +1,107 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : vertx-micrometer-metrics
+# Version       : 3.9.7
+# Source repo   : https://github.com/vert-x3/vertx-micrometer-metrics
+# Tested on     : RHEL UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer's  : Vikas Gupta <vikas.gupta8@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+PACKAGE_NAME=vertx-micrometer-metrics
+PACKAGE_VERSION=3.9.7
+PACKAGE_URL=https://github.com/vert-x3/vertx-micrometer-metrics
+
+mkdir -p /home/tester
+export HOME=/home/tester
```

Comment:
It's not a good idea to override the `HOME` env. Please use a different name.

Reply:
renamed to HOME_DIR